docs/modding/workshop/workshopgui.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Workshop Uploader GUI
+---
+
```

Comment:
Can you add something like this here:
```
# Workshop Uploader

The workshop uploader is a GUI frontend that can upload addons to the Steam workshop.
Addons may contain maps, models, textures, materials and anything else that can be put in a normal Source Mod.
```

Just needs to be a simple summary.

Reply:
also added in recent commit